spec/controllers/miq_ae_tools_controller_spec.rb

```diff
@@ -318,7 +321,6 @@
         end
 
         before do
-          allow(GitRepository).to receive(:create).with(:url => git_url).and_return(git_repo)
```

Comment:
I think this could've been left in and adjusted to include the `:verify_ssl => verify_ssl` part, so the new lines 333, 366, and 377 can go away.